public/editor-client/src/api/index.ts

```diff
@@ -848,3 +848,40 @@ export const updateBlockScreenshot = async ({
 };
 
 //#endregion
+
+//#region FiltersFields
+
+export const getFields = async (data: {
+  postId: string;
+  loopAttributes: string;
+}) => {
+  try {
+    const config = getConfig();
+
+    if (!config) {
+      throw new Error(t("Invalid __BRZ_PLUGIN_ENV__"));
+    }
+
+    const { editorVersion, url, hash, actions } = config;
+
+    const body = new URLSearchParams({
+      hash: hash,
+      version: editorVersion,
+      action: actions.filterFields ?? "brizy_filter_fields",
```

Comment:
theoretically we don't need `?? "brizy_filter_fields"` because if your action will be missing in config, then config reader will throw an error